gno.land/Makefile

```
@@ -6,12 +6,13 @@ help:
 rundep=go run -modfile ../misc/devdeps/go.mod
 
 .PHONY: build
-build: build.gnoland build.gnokey build.gnoweb build.gnofaucet
+build: build.gnoland build.gnokey build.gnoweb build.gnofaucet _build.tm2txsync
```

```suggestion
build: build.gnoland build.gnokey build.gnoweb build.gnofaucet _build.gnotxsync
```

Should be renamed to `gnotxsync` too.

---

good catch. I updated them all.